### PR TITLE
Update apps.yaml

### PR DESCRIPTION
**Pull request by fuzhanrahmanian**
Added MADAP to the app list.

### CODE REVIEW

**Comment by tsthakur** [comment]
Please add acknowledgement to BIG-MAP in your readme file by using the exact phrasing mentioned in the instructions here - https://github.com/BIG-MAP/big-map-registry#adding-an-app-to-the-registry

File: apps.yaml [hunk 1]
```diff
@@ -13,6 +13,22 @@
...
+    metadata:
+        authors: Fuzhan Rahmanian
+        description: |
+            Modular and Autonomous Data Analysis Platform (MADAP) is a well-documented python package which can be used for electrochmeical data analysis for three different classes: Impedance, Arrhenius and Voltammetry
+        documentation_url: https://fuzhanrahmanian.github.io/MADAP/
+        state: stable
+        title: MADAP
+        external_url: https://youtu.be/nL-eJpb1AxI
+        version: 0.11.0
+        logo: https://github.com/fuzhanrahmanian/MADAP/blob/master/logo.png
```
Comment (tsthakur) [inline apps.yaml:31]: Please use the raw image url like following - https://raw.githubusercontent.com/fuzhanrahmanian/MADAP/master/logo.png

**Review by tsthakur — CHANGES_REQUESTED** [review]

**Comment by tsthakur** [comment]
Thank you for the prompt changes, but there's a typo in you readme, the following subsection should be called Acknowledgement https://github.com/fuzhanrahmanian/MADAP#references-1

**Review by tsthakur — APPROVED** [review]

**Comment by fuzhanrahmanian** [comment]
> Thank you for the prompt changes, but there's a typo in you readme, the following subsection should be called Acknowledgement https://github.com/fuzhanrahmanian/MADAP#references-1

Sorry, It is corrected now.

**Review by tsthakur — APPROVED** [review]

**Pull request merged**